modal/mount.py

```diff
@@ -155,31 +156,30 @@ def top_level_paths(self) -> list[tuple[Path, PurePosixPath]]:
         return [(self.local_dir, self.remote_path)]
 
 
-def module_mount_condition(module_base: Path):
+def module_mount_ignore_condition(module_base: Path) -> Callable[[Path], bool]:
```

Comment:
Just in case someone is using this (it's public after all) it might be better to create a new one than modifying the existing one